src/orquestra/quantum/operators/_utils.py

```diff
@@ -192,7 +191,7 @@ def generate_random_pauliop(
 
 
 def evaluate_operator(
-    operator: PauliRepresentation, expectation_values: ExpectationValues
+    operator: PauliRepresentation, expectation_values
```

Comment:
Why removing type here?

Reply:
circular imports :/

Reply:
There is probably a better way to get around this issue?

Reply:
Ok, then this is a symptom of some more fundamental issues – please create a story for that 🙏

Reply:
Done! ([ZQS-1157](https://zapatacomputing.atlassian.net/browse/ZQS-1157))